wiki/en/Running-a-Server.md

```diff
@@ -229,26 +229,27 @@ Most common functions in Jamulus can be set using the GUI, but these and others
 For example on Windows, to use a specific settings file, right-click on the Jamulus shortcut and choose "Properties" > Target. Add the necessary arguments to Jamulus.exe:
 
 ```shell
-"C:\Program Files\Jamulus\Jamulus.exe" --serverbindip 192.168.0.100 
+"C:\Program Files\Jamulus\Jamulus.exe" --serverbindip 192.168.0.100
 ```
 
 For macOS, start a Terminal window and run Jamulus with the desired options like this:
 
 ```shell
- /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100 
+ /Applications/Jamulus.app/Contents/MacOS/Jamulus --serverbindip 192.168.0.100
 ```
 
 **Note**: Command-line options will not alter settings made in the GUI.
 
 - `-d`  or `--discononquit`   Disconnect all clients on quit. Normally, when a server is stopped or restarted, any clients that have not used their "Disconnect" buttons will re-establish connection when the server comes back up again. Using this option forces clients to manually re-establish their connections to the server.  
-- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.  
+- `-e`  or `--directoryserver`  Make the server public and sets its genre (see also `-o`). See [server types](#server-types) for further information.
+- `--directoryfile` Enable directory to remember registered servers even if the directory is restarted. You'll need to specify a path to an existing and writable file in which server information is stored.
```

Comment:
That's the important one...

Reply:
> existing and

I fixed that bit :)  -- it creates the file if not present.

Reply:
Changed.